addons/addon-base-raas/packages/base-raas-services/lib/study/study-service.js

```diff
@@ -388,6 +401,13 @@ class StudyService extends Service {
     // return auditWriterService.write(requestContext, auditEvent);
     return auditWriterService.writeAndForget(requestContext, auditEvent);
   }
+
+  // ensure that study accessType is read/write for Open Data category
+  validateStudyType(accessType, studyCategory) {
```

Comment:
Probably could have a better name, since it's only checking if accessType is ReadWrite. Maybe something like validateReadWrite() ? 

If you're planning to reuse this for other accessType checks in the future, I'll ignore this.

Reply:
Yeah, I just created a validation method where we can dump every thing. The validation turned out to be pretty straight forward for now- planning on leaving it as it is unless you have strong objections.